.github/workflows/ci.yml

```diff
@@ -21,15 +21,14 @@ jobs:
         python-version: ["3.9", "3.13"]
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: ${{ matrix.python-version }}
       - uses: astral-sh/setup-uv@v4
         with:
+          python-version: ${{ matrix.python-version }}
           cache-dependency-glob: pyproject.toml
       - run: uv tool install hatch
       - run: hatch test --cover --python=${{ matrix.python-version }}
+      - run: hatch --env hatch-test.py${{ matrix.python-version }} run coverage xml
```

Comment:
I assume there is a better way to achieve this, but this works.

Reply:
sure does! I don’t know a more elegant way.